src/components/calculator.js

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
```

Comment:
- [ ] As required for the project, kindly ensure that the `Calculator.js` component a function based component and not a class based.

Reply:
I have refactored from class component to function component